cmd/create/breakglasscredential/cmd.go

```diff
@@ -0,0 +1,80 @@
+package breakglasscredential
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/breakglasscredential"
+	"github.com/openshift/rosa/pkg/externalauthprovider"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var breakGlassCredentialArgs *breakglasscredential.BreakGlassCredentialArgs
+
+var Cmd = &cobra.Command{
+	Use:     "break-glass-credential",
+	Aliases: []string{"break-glass-credentials", "breakglasscredential", "breakglasscredentials"},
+	Short:   "Create a break glass credential for a cluster.",
+	Long:    "Create a break glass credential for a hosted control plane cluster with external authentication enabled.",
+	Example: `  # Interactively create a break glass credential to a cluster named "mycluster"
+  rosa create break-glass-credential --cluster=mycluster --interactive`,
+	Run:    run,
+	Hidden: true,
+	Args:   cobra.NoArgs,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	ocm.AddClusterFlag(Cmd)
+
+	interactive.AddFlag(flags)
```

Comment:
@chenz4027 You could just pass `Cmd.Flags()` here rather than assigning the `flags` variable. Not a biggie.